arkouda_proxy_client/arkouda_proxy_client/client.py

```diff
@@ -95,8 +445,16 @@ def disconnect(self) -> None:
                         help='number of args, defaults to -1')
     parser.add_argument('--args', type=str,
                         help='space-delimited list of args for the cmd')
+    parser.add_argument('--request_id', type=str,
+                        help='request_id for the asnyc request')
+    parser.add_argument('--channel_type', type=str,
+                        help='client type for the asnyc request')    
+    
 
     args = parser.parse_args()
 
     channel = GrpcChannel(connect_url=args.arkouda_proxy_url, user=args.user)
     channel.send_string_message(args.cmd, recv_binary=False, args=args.args, size=args.size)
+
+    #channel = PollingGrpcChannel(connect_url=args.arkouda_proxy_url, user=args.user)
+    #channel.send_string_message(args.cmd, recv_binary=False, args=args.args, size=args.size)
```

Comment:
do we wanna keep this commented out lines?
```suggestion
```

Reply:
@pierce314159 removed!

Reply:
all set

Reply:
@pierce314159 removed, all set!

Reply:
@pierce314159 fixed!